src/media/js/buttons.js

```diff
@@ -23,14 +23,14 @@ define('buttons',
         markBtnsAsUninstalled();
     });
 
-    function setInstallBtnState($button, text, cls) {
+    function setInstallBtnState($button, css_class) {
         // Sets install button state (its text and its classes, which
         // currently determines its click handler).
-        revertButton($button, text);
-        $button.addClass(cls);
+        revertButton($button);
+        $button.addClass(css_class);
     }
 
-    function revertButton($button, text) {
```

Comment:
Since `revertButton()` was not using `text`, I removed it. For the same reason I removed it from `setInstallBtnState()` which was just passing it down to `revertButton()`.